cli/cook/version.py

```diff
@@ -1 +1 @@
-VERSION = '0.1.2a2'
+VERSION = '0.1.2a3'
```

Comment:
This is a new feature.  Typically we should be bumping the minor version, although maybe we had something else in mind.  What does the "a" stand for?

Reply:
Changed to `0.2.0`. The "a" was for alpha, but I ditched it